drivers/storage/ide/uniata/id_ata.cpp

```diff
@@ -10754,11 +10754,19 @@ DriverEntry(
     }
 
     if(WinVer_Id() >= WinVer_2k) {
+#ifndef __REACTOS__
         if(AtapiRegCheckParameterValue(NULL, L"Paramaters\\PnpInterface", L"1", 0)) {
+#else
+        if(AtapiRegCheckParameterValue(NULL, L"Parameters\\PnpInterface", L"1", 0)) {
+#endif
```

Comment:
Is it worth enclosing this part of code in ifndef cases just for a typo? Can't be that sent upstream?

Reply:
Yes. As far as I know, there is no upstream repository, and @Alter-1 has been inactive for quite a while, sadly.